### PR TITLE
Feat: Update plume esx to esx legacy

### PR DESCRIPTION
**Pull request by Hamy-os**
As the repositories plume depends on were moved into one this pr updates the links. Enjoy ;)

### CODE REVIEW

**Comment by Scottywonderful** [comment]
This section of the receipe.yaml requires to be updated as per #9 

```
   # Download ESX resources
   - action: download_github
     src: https://github.com/esx-framework/es_extended
     ref: 694cd6757802c432bd025f8251dee33760080eff
     dest: ./resources/[esx]/es_extended
```

**Comment by Hamy-os** [comment]
> This section of the receipe.yaml requires to be updated as per #9
> 
> ```
>    # Download ESX resources
>    - action: download_github
>      src: https://github.com/esx-framework/es_extended
>      ref: 694cd6757802c432bd025f8251dee33760080eff
>      dest: ./resources/[esx]/es_extended
> ```

this isnt needed as its all included in the esx legacy repo

**Comment by Scottywonderful** [comment]
> > This section of the receipe.yaml requires to be updated as per #9
> > ```
> >    # Download ESX resources
> >    - action: download_github
> >      src: https://github.com/esx-framework/es_extended
> >      ref: 694cd6757802c432bd025f8251dee33760080eff
> >      dest: ./resources/[esx]/es_extended
> > ```
> 
> this isnt needed as its all included in the esx legacy repo

Yeah, after posting this, I notice you removed it in the request and updating it to the [https://github.com/esx-framework/esx-legacy](https://github.com/esx-framework/esx-legacy) which I believe is a great idea being it has been mostly used recently.

File: server.cfg [hunk 1]
```diff
@@ -22,77 +26,89 @@ sets sv_projectDesc "{{recipeDescription}}"
...
+ensure esx_license
+ensure esx_lscustom
+ensure esx_mechanicjob
+ensure esx_multicharacter
+ensure esx_optionalneeds
+ensure esx_phone
+ensure esx_policejob
+ensure esx_property
+ensure esx_realestateagentjob
+ensure
```
Comment (Scottywonderful) [inline server.cfg:109]: This extra ensure is not required.
```suggestion
ensure 
```
Lines 74, 87 and 97 is not needed either, it just becomes a huge gap, like this:

`ensure esx_realestateagentjob
ensure



ensure EasyAdmin
`

I suggest it being shortened to this:
`ensure esx_realestateagentjob
ensure EasyAdmin
`

Comment (Scottywonderful) [reply server.cfg:109]: Hope this helps with this mate :)

**Comment by thelindat** [comment]
The Legacy repo has *everything*, some of which wouldn't be recommended such as
- esx_voice
- esx_phone
- esx_whitelist

Some resources need their config modified as well such as enabling identiy/society integration, or enabling multicharacter in the ESX config.

File: server.cfg [hunk 2]
```diff
@@ -1,91 +1,72 @@
...
-## Extra stuff
-ensure bob74_ipl
-ensure esx_adminplus
+# pma voice config
+setr voice_enableRadioAnim 1
+setr voice_useNativeAudio true
+setr voice_useSendingRangeOnly true
+
+## These resources will start by default.
+ensure [fivem]
```
Comment (tabarra) [inline server.cfg:43]: there is also a resource called "fivem",  name the folder [cfx-default]

File: server.cfg [hunk 3]
```diff
@@ -1,91 +1,72 @@
...
+ensure bob74_ipl
 ensure loadingscreen
 ensure simpledrift
+ensure shotspotter
+ensure mdt
+ensure pma-voice
+ensure rp-radio
+ensure plumehello
+
+# resource cleanup
```
Comment (tabarra) [inline server.cfg:63]: starting just to stop it? i don't like this

File: todo.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Welcome to the plumeESX Legacy todo list :P
```
Comment (tabarra) [inline todo.md:1]: You can put this on the readme, maybe this encourage people to help.

File: recipe.yaml [hunk 5]
```diff
@@ -1,16 +1,17 @@
 $engine: 3
-name: PlumeESX
-version: 2.2.0
-author: Tabarra & Chip
-description: A full featured (13 jobs) and highly configurable yet lightweight ESX v1-final base that can be easily extended. 
+$onesync: legacy
+name: PlumeESXLegacy
+version: 3.0.0
+author: Tabarra, Hamy & Chip
```
Comment (tabarra) [inline recipe.yaml:5]: you can take chip away

File: recipe.yaml [hunk 6]
```diff
@@ -1,16 +1,17 @@
...
+description: A full featured and highly configurable yet lightweight ESX Legacy base that can be easily extended. 
 
 tasks:
   # Download default CFX resources
   - action: download_github
     src: https://github.com/citizenfx/cfx-server-data
     ref: 0169fdddaa9511558cfa17bcf80a832f56c359ff
     subpath: resources
-    dest: ./resources
+    dest: ./resources/[fivem]
```
Comment (tabarra) [inline recipe.yaml:14]: name it `cfx-default`, and update the ref to `b907aa74e2826e363979332ac21b9ad98cd82aa1`

File: recipe.yaml [hunk 7]
```diff
@@ -33,210 +30,108 @@ tasks:
...
-  - action: download_github
-    src: https://github.com/esx-framework/esx_addonaccount
-    ref: 3573919be24068283a3103a08414f64398381513
-    dest: ./resources/[esx]/esx_addonaccount
-
-  - action: download_github
-    src: https://github.com/esx-framework/esx_society
-    ref: b033466e14cfed43742117b11f1526b2e127e241
-    dest: ./resources/[esx]/esx_society
+    src: https://github.com/Hamy-os/plumehello
```
Comment (tabarra) [inline recipe.yaml:57]: no motd plz :(

File: recipe.yaml [hunk 8]
```diff
@@ -33,210 +30,108 @@ tasks:
...
-  - action: download_github
-    src: https://github.com/esx-framework/esx_ambulancejob
-    ref: ba0961095c8a31f1d993c2c13f913dbb37bf52bf
-    dest: ./resources/[esx]/esx_ambulancejob
 
-  - action: download_github
-    src: https://github.com/esx-framework/esx_mechanicjob
-    ref: d0e414fc96d3a07a68cafd57aaff954baa28bf99
-    dest: ./resources/[esx]/esx_mechanicjob
+  - action: move_path
```
Comment (tabarra) [inline recipe.yaml:98]: No need, just use subpath on que action above.

File: recipe.yaml [hunk 9]
```diff
@@ -248,24 +143,90 @@ tasks:
...
 
-  ## KL HUD v2
-  - action: download_github
-    src: https://github.com/Kilichi/Kl_HudV2
-    ref: ac8a321ff335706509bdc6122a2984a80082e669
-    dest: ./resources/Kl_HudV2
+
+  ## make directories
+
+  - action: ensure_dir
```
Comment (tabarra) [inline recipe.yaml:150]: don't need

File: recipe.yaml [hunk 10]
```diff
@@ -33,210 +30,108 @@ tasks:
...
+    ref: 7310453e8a66da9e00a539de271827640b5e735a
+    dest: ./resources/pma-voice
 
   - action: download_github
-    src: https://github.com/esx-framework/esx_garage
-    ref: 6ae703b18965452d148541c061f830469c8a8bd9
-    dest: ./resources/[esx]/esx_garage
+    src: https://github.com/AvarianKnight/rp-radio
+    ref: e9309f7f5ef52f9ad8482f5e55d10da56606a7c9
+    dest: ./tmp/rp-radio
```
Comment (tabarra) [inline recipe.yaml:74]: update reference, Avarian moved to the root path so no need to use the /tmp/

File: recipe.yaml [hunk 11]
```diff
@@ -248,24 +143,90 @@ tasks:
...
+  ## make directories
+
+  - action: ensure_dir
+    path: ./resources/[fivem]
+
+  ## move directories
+
+  - action: move_path
+    src: ./tmp/rp-radio/rp-radio
+    dest: ./resources/rp-radio
```
Comment (tabarra) [inline recipe.yaml:157]: solved by the update above, can delete

File: recipe.yaml [hunk 12]
```diff
@@ -248,24 +143,90 @@ tasks:
...
+
+  ## sql file tweaks
+  - action: replace_string
+    file: ./resources/[legacy]/[esx_addons]/esx_property/esx_property.sql
+    search: 'USE `es_extended`;'
+    replace: ''
+
+  - action: replace_string
+    file: ./resources/[legacy]/[esx_addons]/esx_property/offices.sql
+    search: 'USE `es_extended`;'
```
Comment (tabarra) [inline recipe.yaml:167]: Why we doing this? isn't the big sql file already doing these queries?
(same for the one above)

File: recipe.yaml [hunk 13]
```diff
@@ -248,24 +143,90 @@ tasks:
...
-    search: 'esx_status:onTick'
-    replace: 'Kl_Hud:onTick'
+    file: ./resources/rp-radio/config.lua
+    search: 'Key = 289, -- F2'
+    replace: 'Key = 166, -- F5'
+
 
+  - action: replace_string
+    mode: all_vars
+    file: ./resources/[legacy]/[esx_addons]/esx_multicharacter/server/main.lua
```
Comment (tabarra) [inline recipe.yaml:186]: this accepts a file array, so no need to run the action twice

File: recipe.yaml [hunk 14]
```diff
@@ -248,24 +143,90 @@ tasks:
...
+    
 
-  ## Final touches
   - action: replace_string
     mode: all_vars
     file: ./resources/loadingscreen/config.js
     
+
+  - action: remove_path
+    path: ./tmp
```
Comment (tabarra) [inline recipe.yaml:195]: being removed twice

**Review by tabarra — CHANGES_REQUESTED** [review]
change requests :)

File: loadingscreen/config.js [hunk 15]
```diff
@@ -9,11 +9,11 @@ const config = {
         {key:'SHIFT', desc: 'Drift'},
     ],
```
[the remark this answers is not in the document]
Comment (Hamy-os) [reply loadingscreen/config.js:10]: yeah added a radio keybind

**Comment by Hamy-os** [comment]
alright all changes done

File: loadingscreen/config.js (hunk 15 above)
[the remark this answers is not in the document]
Comment (tabarra) [reply loadingscreen/config.js:10]: Job menu gone?

**Pull request merged**